README.markdown

```diff
@@ -294,13 +294,15 @@ var deviceModels: [String]
 ![Xcode indent settings](screens/indentation.png)
 
 * Method braces and other braces (`if`/`else`/`switch`/`while` etc.) always open on the same line as the statement but close on a new line.
+* `else` should be placed in a new line.
 * Tip: You can re-indent by selecting some code (or **Command-A** to select all) and then **Control-I** (or **Editor ▸ Structure ▸ Re-Indent** in the menu). Some open source code will have 2-space tabs hard coded, so this is a good way to fix that.
 
 **Preferred**:
 ```swift
 if user.isHappy {
     // Do something
-} else {
+} 
+else {
```

Comment:
I added this because there is currently a swiftlint rule for this. I know because I always trigger it... I prefer ray wenderlich's approach:
```swift
if something { 
  // do something
} else {
  // do other things
}
```

But again I don't have an strong opinion and I can adapt. Also I think this kind of rules should be moved from our linter to our future formatter, so we can write it however we want but they reach the repo in its correct format.

Reply:
For better or worse, we do it this way to give a little extra readability in real world code where the control flow can be more complex. With the added benefit it gives extra room for comments about the control flow.